### ISSUE TITLE
2022? 

### ISSUE THREAD

**Issue by ghost** [vulkanprogrammingguide/examples#10]
Where is the code? I bought your book, but i need some source code.

**Comment by doug65536** [comment]
I'm actually deciding not to buy the book because the README says there will be code "shortly" since 2016, and half of the amazon comments are complaining about it.

**Comment by ghost** [comment]
> "shortly" since 2016
Yes, it's a big problem. Now I am using examples by SaschaWillems. (https://github.com/SaschaWillems/Vulkan)
